dotcom-rendering/src/components/Card/Card.tsx

```diff
@@ -515,7 +515,6 @@ export const Card = ({
 	 * Currently pills are only shown within beta containers.
 	 */
 	const showPill =
-		isBetaContainer &&
```

Comment:
I temporarily removed the beta container check so I could view the podcast pill and waveform in the card stories and test the palette changes, but it should be reinstated before merging as we don't want to enable this everywhere yet. It might make sense to add a podcast media card to one of the flexible container stories instead?